.asf.yaml

```diff
@@ -61,6 +61,8 @@ github:
         # contexts are the names of checks that must pass
         contexts:
           - check
+          - lint (3.7)
```

Comment:
Do you think it's worth updating `lint` to `python-lint` as well?

Reply:
yes, good idea. I removed it from this PR, added test-mysql instead. I'll rename lint in a separate pr